### PR TITLE
:arrow_up:  upgrade prettier and fix new issues

### PR DESCRIPTION
**Pull request by MatissJanis**
1. Upgraded `prettier`
2. Auto-fixed the new issues

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "husky": "^3.0.4",
     "npm-run-all": "^4.1.3",
     "patch-package": "^6.1.2",
-    "prettier": "^1.18.1",
+    "prettier": "^2.8.2",
```
Comment (j-f1) [inline package.json:40]: Patch releases of Prettier introduce formatting changes, so it’s recommended to pin an exact version.

```suggestion
    "prettier": "2.8.2",
```

File: packages/loot-design/package.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
     "node-noop": "1.0.0",
     "npm-run-all": "^4.1.3",
     "polished": "^1.8.1",
-    "prettier": "^1.14.2",
+    "prettier": "^2.8.2",
```
Comment (j-f1) [inline packages/loot-design/package.json:27]: Is this dependency needed?

File: packages/desktop-client/src/components/ManageRules.js [hunk 3]
```diff
@@ -157,8 +157,9 @@ export function Value({
             </Text>
           );
         })}
-        {// prettier-ignore
-        numHidden > 0 && (
+        {
+          // prettier-ignore
```
[the remark this answers is not in the document]
Comment (MatissJanis) [reply packages/desktop-client/src/components/ManageRules.js]: Makes sense https://github.com/actualbudget/actual/pull/442/commits/54d2a4a35fb08e0aee3184638ed41e2330a8baef

File: package.json (hunk 1 above)
Comment (MatissJanis) [reply package.json]: Good call-out: https://github.com/actualbudget/actual/pull/442/commits/afdfd3c6b188e160242ba5f8839d177ece53e7f1

File: packages/loot-design/package.json (hunk 2 above)
Comment (MatissJanis) [reply packages/loot-design/package.json]: Nope. Removed this dependency from the individual packages. Now we have it only on the root level.
https://github.com/actualbudget/actual/pull/442/commits/afdfd3c6b188e160242ba5f8839d177ece53e7f1

**Review by trevdor — APPROVED** [review]

**Comment by winklevos** [comment]
@MatissJanis is there a prettier conflict here? 

eslint is suggesting `func()` and will fail to build if it applies. 

image

image

**Pull request merged**